docker/gocd-home/gocd-riemann-notifier.conf

```diff
@@ -0,0 +1 @@
+riemann_host=riemann
```

Comment:
🤔 a good future enhancement here would be actually configuring this through the UI.